django/thunderstore/community/api/experimental/views.py

```diff
@@ -0,0 +1,90 @@
+from collections import OrderedDict
+
+from rest_framework.generics import ListAPIView, get_object_or_404
+from rest_framework.pagination import CursorPagination
+from rest_framework.response import Response
+
+from thunderstore.community.api.experimental.serializers import (
+    CommunitySerializer,
+    PackageCategorySerializer,
+)
+from thunderstore.community.models import Community
+
+
+class CustomCursorPagination(CursorPagination):
+    ordering = "-datetime_created"
+    results_name = "results"
+    page_size = 100
+
+    def get_paginated_response(self, data) -> Response:
```

Comment:
What's the reason behind implementing our own `get_paginated_response` and `get_unpaginated_response`?

Reply:
`get_paginated_response` supports `results_name` and the `get_unpaginated_response` improves consistency as otherwise it would return a list instead of an object (so again to support `results_name`)

Reply:
`get_unpaginated_response` isn't a real method though, DRF doesn't support configuring the response so I had to create my own `list` method on the `ListAPIView` to use it

Reply:
Do we need an unpaginated variant?